lib/utils/cn.ts

```diff
@@ -0,0 +1,6 @@
+import clsx, { ClassValue } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+const cn = (...classNames: ClassValue[]) => twMerge(clsx(classNames));
+
+export { cn };
```

Comment:
I would've also loved for this to be a default export, since it's the it's the only thing we're exporting from the file 😉

```js
export default cn;
```

It not a serious nit, you can obviously just leave it if you wish to.

Reply:
This is purely a preference thing at the end of the day.

My usual convention in this case is that default exports should mostly be used for Components, while the rest can stick with named👍